evap/evaluation/models.py

```diff
@@ -241,6 +241,9 @@ class Course(models.Model, metaclass=LocalizeModelBase):
     # whether the evaluation does take place during the semester, stating that evaluation results will be published while the course is still running
     is_midterm_evaluation = models.BooleanField(verbose_name=_("is midterm evaluation"), default=False)
 
+    # whether the first voter explicitly confirmed that given text answers can be published even if no other person evaluates the course
+    text_results_publish_confirmation = models.BooleanField(verbose_name=_("text results publish confirmation"), default=False)
```

Comment:
what about removing the method `has_enough_voters_to_publish_text_results`, calling this boolean here `has_enough_voters_to_publish_text_results` and setting it to true if the first voter checks the checkbox or the second vote comes in?
advantages: a little less code, no need to access num_voters when checking whether there are enough votes, and no persisted boolean that's completely ignored as soon as a second votes comes in.
disadvantages: migration needs to set it to true for most courses. also, it feels a bit weird to me, probably because it deviates from the structure of the other `has_enough_voters_to_*`, but i see no practical reason to not do it...